### PR TITLE
LOK-2369: Rename service monitor properties to metrics

### PR DESCRIPTION
**Pull request by aurang-zeb313**
## Description
Service Monitor results contains a field named properties which is used to transport metrics gathered from the monitoring poll execution. Rename accordingly.
Starting point is ServiceMonitorResponseImpl. Rename the field properties to metrics. Find all usages of this field in the whole code-base and rename other usages, too. Be aware that this is named metrics in some places already. Check in modules minion, minion-gateway and metric-processor.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2369

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

**Review by cgorantla — APPROVED** [review]
LGTM!

File: minion/plugins/plugins-api/src/main/java/org/opennms/horizon/minion/plugin/api/ServiceMonitorResponseImpl.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public class ServiceMonitorResponseImpl implements ServiceMonitorResponse {
     private String ipAddress;
     private double responseTime;
     private DeviceConfig deviceConfig;
-    private Map<String, Number> properties;
+    private Map<String, Number> metrics;
```
Comment (cgorantla) [inline minion/plugins/plugins-api/src/main/java/org/opennms/horizon/minion/plugin/api/ServiceMonitorResponseImpl.java:37]: May be update this to optionalMetrics

**Review by cgorantla — CHANGES_REQUESTED** [review]

File: minion/plugins/plugins-api/src/main/java/org/opennms/horizon/minion/plugin/api/ServiceMonitorResponseImpl.java (hunk 1 above)
Comment (cgorantla) [reply minion/plugins/plugins-api/src/main/java/org/opennms/horizon/minion/plugin/api/ServiceMonitorResponseImpl.java:37]: Or may be additionalMetrics

Comment (aurang-zeb313) [reply minion/plugins/plugins-api/src/main/java/org/opennms/horizon/minion/plugin/api/ServiceMonitorResponseImpl.java]: OK

File: minion/minion-taskset/taskset-worker/src/main/java/org/opennms/horizon/minion/taskset/worker/impl/TaskExecutionResultProcessorImpl.java [hunk 2]
```diff
@@ -154,9 +154,8 @@ private MonitorResponse formatMonitorResponse(ServiceMonitorResponse smr) {
                 .setReason(Optional.of(smr)
                         .map(ServiceMonitorResponse::getReason)
                         .orElse(MonitorResponse.getDefaultInstance().getReason()))
-                .putAllMetrics(Optional.of(smr)
-                        .map(ServiceMonitorResponse::getProperties)
-                        .orElse(Collections.EMPTY_MAP))
+                .putAllMetrics(
+                        Optional.of(smr).map(ServiceMonitorResponse::getMetrics).orElse(Collections.EMPTY_MAP))
```
Comment (smunir-onms) [inline minion/minion-taskset/taskset-worker/src/main/java/org/opennms/horizon/minion/taskset/worker/impl/TaskExecutionResultProcessorImpl.java:158]: This reference seems unchanged

**Review by fooker — APPROVED** [review]

**Review by cgorantla — APPROVED** [review]
LGTM!

**Pull request merged**